docs/Misc/Discord-Notifier-Basic-Setup.md

```diff
@@ -377,9 +392,9 @@ Scroll up and save
 
 ### Testing
 
-On the top of the integrations page you will now have the ability to test your settings. This only tests the connection between the notifier and your discord. You can use the Test in each *arr and that will test the everything (connection wise).
+On the bottom of all the instruction you will now have the ability to test your settings. This only tests the connection between the notifier and your discord. You can use the Test in each *arr and that will test the everything (connection wise).
```

Comment:
On the bottom of all the instructions pop-ups, you will now have the ability to test your settings for those that have a test available.